src/components/SettingDialog.vue

```diff
@@ -272,14 +341,161 @@ export default defineComponent({
         title: "書き出し先のフォルダを選択",
       });
       if (path) {
-        store.dispatch("SET_SAVING_SETTING_DATA", {
+        store.dispatch("SET_SAVING_SETTING", {
           data: { ...savingSetting.value, fixedExportDir: path },
         });
       }
     };
 
+    const hotkeyRows = computed(() => store.state.hotkeySettings);
+
+    const hotkeyColumns = ref([
+      {
+        name: "action",
+        align: "left",
+        label: "操作",
+        field: "action",
+      },
+      {
+        name: "combination",
+        align: "right",
+        label: "ショートカットキー",
+        field: "combination",
+      },
+    ]);
+
+    let lastAction = ref<string | null>(null);
+    let lastRecord = ref("");
```

Comment:
ホットキー入力中に設定ダイアログを閉じても入力中のままの状態になっていそうです。
将来的に設定画面から色んな画面に遷移することも考えられるので、ホットキーの入力中はホットキーの入力と確定以外の操作ができないようにしてしまうのが安全だと思います。
ダイアログにしましょう！！

---

Even if I close the settings dialog while inputting a hotkey, it seems to remain in the input state.
Since in the future it is possible to transition from the settings dialog to various other screens, I think it would be safer to prevent any operations other than inputting and finalizing a hotkey.
Let's make it a dialog!